samples/GoogleHangoutsChat.gs

```diff
@@ -0,0 +1,90 @@
+var PRIVATE_KEY =
+    '-----BEGIN PRIVATE KEY-----\n...\n-----END PRIVATE KEY-----\n';
+var CLIENT_EMAIL = '...';
+
+/**
+ * Authorizes and makes a request to the Hangouts Chat API
+ * for sending a text message to all DM room bot has been added.
+ * Best before to use service account is to do the Google CodeLabs
```

Comment:
"Best before to use service account is to do" is awkward English. Perhaps "For a more detailed guide on building bots, follow the codelab:"

Reply:
done